tensorflow/compiler/tf2tensorrt/convert/convert_graph.cc

```diff
@@ -408,7 +408,7 @@ Status CreateTRTNode(const ConversionParams& params,
           }
         }
 
-        // Rewrire data input if it's not found in original graph.
```

Comment:
Thank you for catching this typo! Would you please split this to another PR, to avoid mixing changes to two different components into one PR?

Reply:
Are you sure this wasn't meant to be "rewire"?

Reply:
It is "rewire", it was "rewrire".

Reply:
Fixed.

Reply:
Would you please split this change to another PR? There are reasons for this: (1) the PR description doesn't mentioned this change. (2) changes to tf2tensorrt has a more complicated process for merging and can't be merged automatically.  (3) allow "owners" to the two components to approve changes for each component.

Reply:
Sure. Typo fix is reverted.